docs/source/guide/predictions.md

```diff
@@ -640,6 +655,326 @@ from label_studio_converter import brush
+               "from_name":"bbox",
+               "to_name":"image",
+               "type":"rectangle"
+            },
+            {
+               "original_width":864,
+               "original_height":1296,
+               "image_rotation":0,
+               "value":{
+                  "x":72.26666666666667,
+                  "y":59.46666666666667,
+                  "width":11.866666666666667,
+                  "height":4.444444444444445,
+                  "rotation":0,
+                  "labels":[
+                     "Price"
+                  ]
+               },
+               "id":"bb2",
+               "from_name":"label",
+               "to_name":"image",
+               "type":"labels"
+            },
+            {
+               "original_width":864,
+               "original_height":1296,
+               "image_rotation":0,
+               "value":{
+                  "x":72.26666666666667,
+                  "y":59.46666666666667,
+                  "width":11.866666666666667,
+                  "height":4.444444444444445,
+                  "rotation":0,
+                  "text":[
+                     "50.000"
+                  ]
+               },
+               "id":"bb2",
+               "from_name":"transcription",
+               "to_name":"image",
+               "type":"textarea"
+            },
+            {
+               "original_width":864,
+               "original_height":1296,
+               "image_rotation":0,
+               "value":{
+                  "x":48.93333333333333,
+                  "y":61.333333333333336,
+                  "width":9.733333333333333,
+                  "height":2.8444444444444446,
+                  "rotation":0
+               },
+               "id":"bb1",
+               "from_name":"bbox",
+               "to_name":"image",
+               "type":"rectangle"
+            },
+            {
+               "original_width":864,
+               "original_height":1296,
+               "image_rotation":0,
+               "value":{
+                  "x":48.93333333333333,
+                  "y":61.333333333333336,
+                  "width":9.733333333333333,
+                  "height":2.8444444444444446,
+                  "rotation":0,
+                  "labels":[
+                     "Text"
+                  ]
+               },
+               "id":"bb1",
+               "from_name":"label",
+               "to_name":"image",
+               "type":"labels"
+            },
+            {
+               "original_width":864,
+               "original_height":1296,
+               "image_rotation":0,
+               "value":{
+                  "x":48.93333333333333,
+                  "y":61.333333333333336,
+                  "width":9.733333333333333,
+                  "height":2.8444444444444446,
+                  "rotation":0,
+                  "text":[
+                     "TOTAL"
+                  ]
+               },
+               "id":"bb1",
```

Comment:
Looks like very long example. Could we truncate the result array?

Reply:
yeah I can remove the extra bboxes so it's an example with just one bbox + label + textarea example :)